modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSQuickActions.testcase

```diff
@@ -65,6 +65,42 @@ definition {
 		}
 	}
 
+	@description = "LPS-155455: When hovering over the first line item and the quick action menu is displayed on the 1st line"
+	@priority = "5"
+	test CanDisplayOnHover {
+		task ("Then the quick action menu is displayed on the 1st line") {
+			FDSQuickActions.viewQuickActionMenuIsPresent(title = "Sample1");
+		}
+	}
+
+	@description = "LPS-155459: When clicking on the ellipsis and hovering over another row, multiple quick action menu are displayed"
+	@priority = "3"
+	test CanDisplayOnMultipleActiveRows {
+		task ("When click on Elipsis button 1st") {
+			Click(
+				key_title = "Sample1",
+				locator1 = "FrontendDataSet#VERTICAL_ELLIPSIS_BUTTON");
+		}
+
+		task ("Then the quick action menu is displayed on the 1st line") {
+			AssertElementPresent(locator1 = "FrontendDataSet#DROPDOWN_MENU");
+		}
+
+		task ("When hovering over the 2nd line item") {
+			MouseOver(
+				key_itemName = "Sample2",
+				locator1 = "FrontendDataSet#TABLE_ITEM_ROW");
+		}
+
+		task ("Then the quick action menu is displayed on the 2nd line") {
+			FDSQuickActions.viewQuickActionMenuIsPresent(title = "Sample2");
+		}
+
+		task ("Then the quick action menu is displayed on the 1st line") {
+			AssertElementPresent(locator1 = "FrontendDataSet#DROPDOWN_MENU");
```

Comment:
same comment

Reply:
@john-co I thought was to confirm that both menus (blue and red) were being shown. That's why I used the dropdown and not the `FDSQuickActions.viewQuickActionMenuIsPresent `macro. So I confused and would only need to confirm that the quick menu (blue) is visible?
![FDS quick menu](https://user-images.githubusercontent.com/44584050/175972891-1719cdca-6878-4fca-8f84-1f6110302294.jpg)